### PR TITLE
More obs ...

### LINKED ISSUE #885: More marine obs as part of the default list
Add these back to the possible obs spaces:
```
	sst_abi_g16_l3c.yaml
	sst_abi_g17_l3c.yaml
	sst_ahi_h08_l3c.yaml
	sst_metopa_l3u.yaml
	sst_metopc_l3u.yaml
	sst_viirs_n20_l3u.yaml
	sst_viirs_npp_l3u.yaml
```
and test

### PR DESCRIPTION
**Pull request by guillaumevernieres**
- fixes #885

### CODE REVIEW

**Comment by guillaumevernieres** [comment]
It's a lot of copy/pasting. I wonder if we should have an application that generates template yaml files for these sst retrieval ... humm.

**Review by apchoiCMD — DISMISSED** [review]

File: parm/soca/obs/config/sst_metopa_l3u.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+obs space:
+  name: sst_metopa_l3u
```
Comment (ShastriPaturi) [inline parm/soca/obs/config/sst_metopa_l3u.yaml]: @guillaumevernieres, is it worth renaming this as: `sst_avhrr_metopa_l3u`, similar to `sst_abi_g16_l3c` and `sst_viirs_n20_l3u`, to follow a pattern?

File: parm/soca/obs/config/sst_metopc_l3u.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+obs space:
+  name: sst_metopc_l3u
```
Comment (ShastriPaturi) [inline parm/soca/obs/config/sst_metopc_l3u.yaml]: @guillaumevernieres, same as above

File: parm/soca/obs/obs_list.yaml [hunk 3]
```diff
@@ -1,5 +1,17 @@
 observers:
+# ADT
 - !INC ${OBS_YAML_DIR}/adt_rads_all.yaml
+
+# SST
+- !INC ${OBS_YAML_DIR}/sst_metopa_l3u.yaml
```
Comment (ShastriPaturi) [inline parm/soca/obs/obs_list.yaml]: @guillaumevernieres, if you concur with the previous naming suggestion, these need to be changed and `metop<b,c>` as well.

File: parm/soca/obsprep/obsprep_config.yaml [hunk 4]
```diff
@@ -32,6 +37,22 @@ observations:
     dmpdir subdir: ocean/icec
     output file: icec_amsr2_south.ioda.nc
     dmpdir regex: AMSR2-SEAICE-SH_v2r2_GW1_s*.nc
+
+# SST
+- obs space:
+    name: sst_metopa_l3u
```
Comment (ShastriPaturi) [inline parm/soca/obsprep/obsprep_config.yaml]: Here too

File: parm/soca/obsprep/obsprep_config.yaml [hunk 5]
```diff
@@ -32,6 +37,22 @@ observations:
...
+    provider: GHRSST
+    dmpdir subdir: 'ocean/sst'
+    output file: sst_metopa_l3u.ioda.nc
```
Comment (ShastriPaturi) [inline parm/soca/obsprep/obsprep_config.yaml]: Maybe the name here as well.
Suggested change: `sst_avhrr_metopa_l3u.ioda.nc`

**Review by ShastriPaturi — CHANGES_REQUESTED** [review]

**Comment by AndrewEichmann-NOAA** [comment]
> It's a lot of copy/pasting. I wonder if we should have an application that generates template yaml files for these sst retrieval ... humm.

How often are we going to be doing this?

File: parm/soca/obs/config/sst_metopa_l3u.yaml (hunk 1 above)
Comment (AndrewEichmann-NOAA) [reply parm/soca/obs/config/sst_metopa_l3u.yaml]: I like this concept

**Review by AndrewEichmann-NOAA — DISMISSED** [review]
looks right

File: parm/soca/obs/config/sst_metopa_l3u.yaml (hunk 1 above)
Comment (guillaumevernieres) [reply parm/soca/obs/config/sst_metopa_l3u.yaml]: I agree @ShastriPaturi , it should be renamed.

**Review by ShastriPaturi — DISMISSED** [review]
Thanks for the changes @guillaumevernieres.

**Review by apchoiCMD — DISMISSED** [review]
Looks right-

**Review by AndrewEichmann-NOAA — DISMISSED** [review]

**Review by apchoiCMD — DISMISSED** [review]

**Review by apchoiCMD — APPROVED** [review]

**Review by AndrewEichmann-NOAA — APPROVED** [review]
So no window for the SSTs?

File: parm/soca/obsprep/obsprep_config.yaml [hunk 6]
```diff
@@ -31,19 +36,116 @@ observations:
...
+    output file: sst_avhrr_mb_l3u.ioda.nc
     dmpdir regex: '*-L3U_GHRSST-SSTsubskin-AVHRRF_MB-ACSPO_V*.nc'
     bounds:
       units: C
       min: -3.0
       max: 50.0
-    window:
-       back: 4
-       forward: 0   
     binning:
```
Comment (ShastriPaturi) [inline parm/soca/obsprep/obsprep_config.yaml:65]: @guillaumevernieres any reason why the `overlapping window` entries are not considered here?

**Review by ShastriPaturi — CHANGES_REQUESTED** [review]

File: parm/soca/obsprep/obsprep_config.yaml (hunk 6 above)
Comment (guillaumevernieres) [reply parm/soca/obsprep/obsprep_config.yaml:65]: ... Look up ... it says sst @ShastriPaturi . We're certainly not doing overlapping windows for sst's!!!

**Review by ShastriPaturi — APPROVED** [review]

**Comment by guillaumevernieres** [comment]
> So no window for the SSTs?

Yeah ... HELL NO @AndrewEichmann-NOAA :
1 - It's affecting the mixed layer, time scales of interests in 10's of minutes, we don't want to play gods with redating that obs type!
2 - have you seen how dense these obs are???

**Comment by ShastriPaturi** [comment]
> So no window for the SSTs?

No... we should be having SST obs for the gdas cycle. I asked an incorrect question.

**Comment by RussTreadon-NOAA** [comment]
@guillaumevernieres , this PR modified the submodule hashes for the following JEDI repos
- [fv3-jedi @ b4bc841](https://github.com/jcsda/fv3-jedi/tree/b4bc841922609c28457e0899a397d001508a34de) (1/8/2024)
- [gsibec @ e760ec7](https://github.com/GEOS-ESM/GSIbec/tree/e760ec73c117d1456fd6f7ead6bf4c7f4c38e139) (4/17/2023)
- [ioda @ f395e1d](https://github.com/jcsda/ioda/tree/f395e1d33b3c6ce7b970da25f589d5fd92f5197b) (1/10/2024)
- [iodaconv @ 01adb65](https://github.com/JCSDA-internal/ioda-converters/tree/01adb65f60dcbb7e83b50d25a82d81e915bdcb6d) (1/9/2024)
- [land-imsproc @ 6373819](https://github.com/NOAA-PSL/land-IMS_proc/tree/6373819ca034d66523cb7852cd7b1b66f3f8ae07) (12/7/2022)
- [oops @ 60f9392](https://github.com/jcsda/oops/tree/60f93924fe446714fcb04d96f6930a760db74b23) (1/8/2024)
- [saber @ 4f1956c](https://github.com/jcsda/saber/tree/4f1956c38267200feff91a8e4231e82df3885637) (1/5/2024)
- [soca @ 44e1a51](https://github.com/jcsda-internal/soca/tree/44e1a5144ab2682f739af976feff67a537a2f164) (1/16/2024)
- [ufo @ e41a1c9](https://github.com/jcsda/ufo/tree/e41a1c928150944795ed1276c84c4f1e37a47c99) (1/4/2024)

The date is in `( )` on each line is the commit date for the given JEDI repo hash.  

Here are the hashes we had in GDASApp `develop` at [`446ffe7`](https://github.com/NOAA-EMC/GDASApp/commit/446ffe72c3a024e97320c107f31fd357a41c868b).  This is the GDASApp `develop` before PR #887 was merged into `develop`.

- [fv3-jedi @ 5e4679a](https://github.com/jcsda/fv3-jedi/tree/5e4679a3138a46e8de37b91bc39988e4dbf06499) (1/23/2024)
- [gsibec @ 17823ae](https://github.com/GEOS-ESM/GSIbec/tree/17823aef706e5d69e373e4c4c5dfadac99d5b800) (4/27/2023)
- [ioda @ 79f9659](https://github.com/jcsda/ioda/tree/79f96592d6cdb5fa5b7696735d7ff4296d583d88) (1/19/2024)
- [iodaconv @ 18094c0](https://github.com/JCSDA-internal/ioda-converters/tree/18094c03f0ed2d2468b6fc974bfe4a6798aa1025) (1/24/2024)
- [land-imsproc @ 2ac7c0d](https://github.com/NOAA-PSL/land-IMS_proc/tree/2ac7c0d90d3ebd449ad11bce155a71d381b4534a) (5/19/2023)
- [oops @ 61fad29](https://github.com/jcsda/oops/tree/61fad29cc6924f07a28d4d878d3ffe3b77f7f674) (1/24/2024)
- [saber @ c902217](https://github.com/jcsda/saber/tree/c9022178972cef1c30995473f64d46a232c0cb88) (1/18/2024)
- [soca @ 9a29b48](https://github.com/jcsda-internal/soca/tree/9a29b48d6e035034d2205066a688af5e858d08c0) (1/18/2024)
- [ufo @ 3d320d7](https://github.com/jcsda/ufo/tree/3d320d7d276e6fc6670dd837bf27e69a91ab664c) (1/24/2024)

GDASApp at [`446ffe7`](https://github.com/NOAA-EMC/GDASApp/commit/446ffe72c3a024e97320c107f31fd357a41c868b) pointed at newer JEDI repo hashes.   

Why do we need to move the submodule hashes back in time with PR #887?

**Comment by guillaumevernieres** [comment]
> Why do we need to move the submodule hashes back in time with PR #887?

We don't @RussTreadon-NOAA , my mistake ... Although I see 3 green checks on the review side! 

I'll submit a PR reverting to what it was before.

**Pull request merged**